packages/server/src/channels/smooch/conduit.ts

```diff
@@ -62,9 +62,24 @@ export class SmoochConduit extends ConduitInstance<SmoochConfig, SmoochContext>
     return SmoochSenders
   }
 
-  public async extractEndpoint(payload: { context: SmoochPayload; message: SmoochMessage }): Promise<EndpointContent> {
+  public async extractEndpoint(payload: {
+    context: SmoochPayload
+    message: SmoochMessage
+    action: SmoochAction
+  }): Promise<EndpointContent> {
+    const postback = payload.message.action?.payload
+    let content
+
+    if (postback?.startsWith('say::')) {
```

Comment:
`say::` and `postback::` should be moved into constants somewhere. If they could be used in other channels, maybe put them in the base channel.